src/promql/src/planner.rs

```diff
@@ -74,6 +77,10 @@ const DEFAULT_FIELD_COLUMN: &str = "value";
 /// Special modifier to project field columns under multi-field mode
 const FIELD_COLUMN_MATCHER: &str = "__field__";
 
+/// default value column name for prometheus metrics.
+/// This is the same with the constant defined in `servers`
+const GREPTIME_VALUE: &str = "greptime_value";
```

Comment:
It would be better to move it into `common-base`, so we can avoid this duplicated definition.